release.sh

```diff
@@ -98,56 +103,86 @@ git checkout "$BRANCH"
 git pull
 git checkout --detach
 
-# Edit package.json and bower.json to the right version (see
+# Edit package.json to the right version (see
 # http://stackoverflow.com/a/22084103 for why we need the .bak file to make
 # this mac & linux compatible)
 sed -i.bak -E 's|"version": "[^"]+",|"version": "'$VERSION'",|' package.json
 rm -f package.json.bak
 
-# Build generated files and add them to the repository (for bower)
+# Build generated files and add them to the repository
 git clean -fdx dist
 yarn dist
 sed -i.bak -E '/^\/dist\/$/d' .gitignore
 rm -f .gitignore.bak
 git add .gitignore dist/
 
-# Update the version number in CDN URLs included in the README files,
+# Edit docs to use CSS from CDN (#1520)
+# grep -l '{@stylesheet: static/katex.min.css}' docs/*.md | xargs sed -i.bak \
+#     's|{@stylesheet: static/katex.min.css}|<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@./dist/katex.min.css" integrity="sha256-katex.min.css" crossorigin="anonymous"/>|'
+
+# Update the version number in CDN URLs included in the README and the documentation,
 # and regenerate the Subresource Integrity hash for these files.
-node update-sri.js "${VERSION}" README.md contrib/*/README.md dist/README.md
+node update-sri.js "${VERSION}" README.md contrib/*/README.md dist/README.md \
+    docs/*.md docs/*.md.bak website/pages/index.html
```

Comment:
Why `docs/*.md.bak`? Are these actually checked in?

Reply:
@edemaine They will be restored after `yarn version`, which clones the current documentation.